gen/suites/actor_creation/illegal_actor.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+
+	"github.com/filecoin-project/test-vectors/chaos"
+	. "github.com/filecoin-project/test-vectors/gen/builders"
+)
+
+func actorAbortWithSystemExitCode(v *Builder) {
+	v.Messages.SetDefaults(GasLimit(1_000_000_000), GasPremium(1), GasFeeCap(200))
+
+	alice := v.Actors.Account(address.SECP256K1, abi.NewTokenAmount(1_000_000_000_000))
+	v.CommitPreconditions()
+
+	var msgs []*ApplicableMessage
+	//for i := 0; i <= 15; i++ {
+	for i := 1; i <= 15; i++ {
```

Comment:
Can we use `exitcode.FirstActorErrorCode` with a `lesser than` comparison here? I think it will make it more self-descriptive.